src/Engine/Manager/PhysicsManager.hpp

```diff
@@ -219,6 +219,15 @@ class PhysicsManager {
          */
         ENGINE_API void MakeDynamic(Component::RigidBody* comp);
 
+        /// Enables/disables ghost functionality on a rigid body.
+        /**
+         * @param comp Rigid body to alter state of.
+         * @param ghost True: makes the rigid object a ghost, disregarding all
+         collisions. False: disables ghost state, reverting to kinematic or
```

Comment:
This and the row below should start with a * so that the text is aligned with the text above.

Reply:
It's the same line, just wrapped. Will the asterisks not interfere with the doxygen output?